### PR TITLE
Adds alt-click to spraycans

### PR DESCRIPTION
**Pull request by coiax**
:cl: coiax
rscadd: Alt-clicking a spraycan toggles the cap.
/:cl:

### CODE REVIEW

**Comment by WJohn** [comment]
Why not just remove the cap entirely? What purpose does it serve? Is there a time when you do NOT want to spray a wall you click on with the spray can in your hand?

File: code/game/objects/items/crayons.dm [hunk 1]
```diff
@@ -497,7 +504,8 @@
 	paint_color = null
 
 	item_state = "spraycan"
-	desc = "A metallic container containing tasty paint."
+	desc = "A metallic container containing tasty paint.\n\
+		Alt-click to toggle the cap."
```
Comment (PKPenguin321) [inline code/game/objects/items/crayons.dm]: i'd kinda prefer if you did this like i did in https://github.com/tgstation/tgstation/pull/17050 for the sake of consistency

**Comment by WJohn** [comment]
Hell, just turn the spray can into a child of rainbow crayons, with extra uses. It has effectively all the same functions, so there's no reason that it should require you to go through 2 UIs just to set things how you want.

**Comment by coiax** [comment]
1) Spraycans are already crayons.
2) I already updated the UI so it's not so awful anymore.

**Pull request merged**